codec-mqtt/src/main/java/io/netty/handler/codec/mqtt/MqttUnsubscribeMessage.java

```diff
@@ -22,11 +22,14 @@
  */
 public final class MqttUnsubscribeMessage extends MqttMessage {
 
+    private final MqttUnsubscribePayload payload;
+
     public MqttUnsubscribeMessage(
             MqttFixedHeader mqttFixedHeader,
             MqttMessageIdVariableHeader variableHeader,
             MqttUnsubscribePayload payload) {
-        super(mqttFixedHeader, variableHeader, payload);
+        super(mqttFixedHeader, variableHeader);
+        this.payload = payload;
```

Comment:
@normanmaurer I think better way to throw exception in `MqttDecoder.decodePayload`

here 

```
            default:
                // unknown payload , no byte consumed
                return new Result<Object>(null, 0);
```